ledger/recovering-indexer-integration-tests/src/test/suite/scala/com/digitalasset/platform/indexer/RecoveringIndexerIntegrationSpec.scala

```diff
@@ -230,9 +230,12 @@ object RecoveringIndexerIntegrationSpec {
         logCtx: LoggingContext
     ): ResourceOwner[ParticipantState] = {
       val metricRegistry = new MetricRegistry
-      new InMemoryLedgerReaderWriter.SingleParticipantOwner(ledgerId, participantId, metricRegistry)
-        .map(readerWriter =>
-          new KeyValueParticipantState(readerWriter, readerWriter, metricRegistry))
+      new InMemoryLedgerReaderWriter.SingleParticipantOwner(
+        ledgerId,
+        participantId,
+        metricRegistry = metricRegistry,
```

Comment:
I don't think this adds to readability

Reply:
It's just because there's a default argument in the middle.